src/dune_rules/opam_create.ml

```diff
@@ -281,7 +282,23 @@ let generate project pkg ~template =
     | None -> ""
     | Some (_, s) -> s)
 
-let add_rule sctx ~project ~pkg =
+let add_alias_rule sctx ~project ~pkg =
```

Comment:
If I'm reading this correctly, then we still emit the `.opam` files in the repository root inside the build folder?

Reply:
to answer my own question: we don't emit `*.opam` files in `opam/`, we just expect them to be there.